discover-cpu-partitioning.py

```diff
@@ -92,27 +92,34 @@ def get_pid_cpus_allowed(pid):
 
     return(cpus_allowed)
 
+def output_cpu_info(label, cpu_list):
+    if t_global.args.debug:
+        debug("%s cpus: %d" % (label, len(cpu_list)))
+        short_cpu_list = system_cpu_topology.formatted_cpu_list(cpu_list)
+        formatted_short_cpu_list = ','.join(short_cpu_list)
+        debug("%s cpus: %s" % (label, formatted_short_cpu_list))
+
+    cpu_list = ','.join(map(str, cpu_list))
+    print("%s cpus: %s" % (label, cpu_list))
+
+    print("")
```

Comment:
\n saves you one lline of code

Reply:
I tend to write my code  in preparation for what might happen...as in if I add a line in between the current line and the newline I would have to change the current line if I used \n, by making it a separate line I don't have to touch the current line again...